source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,146 @@
+            {
+                storageConnectionString = sr.ReadToEnd();
+            }
+
+            // Check whether the connection string can be parsed.
+            CloudStorageAccount storageAccount;
+            if (CloudStorageAccount.TryParse(storageConnectionString, out storageAccount))
+            {
+                // If the connection string is valid, proceed with operations against Blob
+                // storage here.
+                // Create the CloudBlobClient that represents the 
+                // Blob storage endpoint for the storage account.
+                CloudBlobClient cloudBlobClient = storageAccount.CreateCloudBlobClient();
+
+                CloudBlobContainer cloudBlobContainer =
+                    cloudBlobClient.GetContainerReference(Constants.ContainerId);
+                await cloudBlobContainer.CreateIfNotExistsAsync();
+
+                // Set the permissions so the blobs are public.
+                BlobContainerPermissions permissions = new BlobContainerPermissions
+                {
+                    PublicAccess = BlobContainerPublicAccessType.Blob
+                };
+
+                await cloudBlobContainer.SetPermissionsAsync(permissions);
+
+                var sourceFile = Path.Combine(Constants.FrameworkPath, Constants.FrameworkName);
+                var blobGuid = Guid.NewGuid().ToString();
+                var cloudFileName = blobGuid + Constants.FrameworkName;
+
+                CloudBlockBlob cloudBlockBlob = cloudBlobContainer.GetBlockBlobReference(cloudFileName);
+
+                await cloudBlockBlob.UploadFromFileAsync(sourceFile);
+
+                BlobContinuationToken blobContinuationToken = null;
+                do
+                {
+                    var results = await cloudBlobContainer.ListBlobsSegmentedAsync(null, blobContinuationToken);
+                    // Get the value of the continuation token returned by the listing call.
+                    blobContinuationToken = results.ContinuationToken;
+                    foreach (IListBlobItem item in results.Results)
+                    {
+                        var uriString = item.Uri.ToString();
+                        if (uriString.Contains(blobGuid))
+                        {
+                            UpdatePodSpec(uriString);
+                        }
+                    }
+                } while (blobContinuationToken != null);
+            }
+            else
+            {
+                throw new InvalidOperationException("invalid connection string");
+            }
+        }
+
+        private static void UpdatePodSpec(string uri)
+        {
+            var sourceFile = Path.Combine(Constants.PodspecPath, Constants.PodspecName);
+            var targetFile = Path.Combine(Constants.TargetPodspecPath, Constants.PodspecName);
+
+            FileInfo fi = new FileInfo(sourceFile);
+            if (!fi.Exists)
+            {
+                throw new FileNotFoundException("missing podspec");
```

Comment:
nit: same aids in debuggability if you include the filename we are complainng about

Reply:
thanks

---
In reply to: [314077399](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r314077399) [](ancestors = 314077399)